src/Serializer/Normalizer/PhoneNumberNormalizer.php

```diff
@@ -11,96 +11,28 @@
 
 namespace Misd\PhoneNumberBundle\Serializer\Normalizer;
 
-use libphonenumber\NumberParseException;
-use libphonenumber\PhoneNumber;
-use libphonenumber\PhoneNumberFormat;
-use libphonenumber\PhoneNumberUtil;
-use Symfony\Component\Serializer\Exception\InvalidArgumentException;
-use Symfony\Component\Serializer\Exception\UnexpectedValueException;
+use Symfony\Component\Serializer\Normalizer\ArrayDenormalizer;
 use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
 use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
 
-/**
- * Phone number serialization for Symfony serializer.
- */
-class PhoneNumberNormalizer implements NormalizerInterface, DenormalizerInterface
-{
-    /**
-     * Region code.
-     *
-     * @var string
-     */
-    private $region;
-
-    /**
-     * Display format.
-     *
-     * @var int
-     */
-    private $format;
-
+if (class_exists(ArrayDenormalizer::class) && !method_exists(ArrayDenormalizer::class, 'setSerializer')) {
```

Comment:
any better idea of detecting `symfony/serializer` >= 6.0?

Reply:
I just took a look and I don't think there is any other choice